src/PowerShellEditorServices/Services/TextDocument/ScriptFile.cs

```diff
@@ -53,6 +52,7 @@ internal sealed class ScriptFile
         /// Gets or sets a boolean that determines whether
         /// semantic analysis should be enabled for this file.
         /// For internal use only.
+        /// TODO: Actually use and respect this property to avoid built-in files from being analyzed.
```

Comment:
And we stopped opening built-ins, so actually `TODO: We can remove this property.`